Tensile/KernelWriterAssembly.py

```diff
@@ -3763,6 +3763,35 @@ def openPersistentLoop(self, kernel):
     if kernel["PersistentKernel"] or kernel["StreamK"]:
       if kernel["StreamK"]:
         # Workload calculations
+        if kernel["StreamKXCCMapping"]:
+          sXCC = self.sgprPool.checkOut(1, "XCC", preventOverflow=0)
+          sGridC = self.sgprPool.checkOut(1, "sGridC", preventOverflow=0)
+          sGridF = self.sgprPool.checkOut(1, "sGridF", preventOverflow=0)
+          sGridM = self.sgprPool.checkOut(1, "sGridM", preventOverflow=0)
+          stmp = self.sgprPool.checkOut(2, "stmp", preventOverflow=0)
```

Comment:
It might be better to set stmp=None if StreamKXCCMapping is power of 2.

Reply:
We provably need to change the assert condition (no need assert for qReg == tmpSgpr == None case)

Reply:
Or, move assert in else part (not power of 2 case)